build.sh

```diff
@@ -8,7 +8,7 @@ CUR_DIR="$(pwd)"
 BUILD_DIR="${CUR_DIR}/.build_${ARCH}"
 ALPINE_VERSION="${ALPINE_VERSION:-latest-stable}"
 ALPINE_REPO="${ALPINE_REPO:-http://dl-cdn.alpinelinux.org/alpine}"
-ROOTFS_ARCHIVE="rootfs.xz.img"
+TOOLBOX_IMAGE="um-update_toolbox.xz.img"
```

Comment:
note to ourselves: when we introduce version numbering, we will likely add that to the image name as well, at which point it makes sense to use
```sh
TOOLBOX_IMAGE="${TOOLBOX_IMAGE}:-um-update_toolbox.xz.img"
```